UnitsNet.Tests/CustomCode/LengthTests.FeetInches.cs

```diff
@@ -41,17 +41,17 @@ public void FeetInchesRoundTrip()
         [InlineData("1′1″", 1.08333333)] // Without space
         [InlineData("1 ft 1 in", 1.08333333)]
         [InlineData("1ft 1in", 1.08333333)]
-        [InlineData( "-1'", -1 )] // Feet only
-        [InlineData( "-1′", -1 )] // Feet only
-        [InlineData( "-1\"", -0.08333333 )] // Inches only
-        [InlineData( "-1″", -0.08333333 )] // Inches only
-        [InlineData( "-1' 1\"", -1.08333333 )] // Normal form
-        [InlineData( "-1′ 1″", -1.08333333 )] // Normal form
-        [InlineData( "- 1′ 1″ ", -1.08333333 )] // Normal form, requires trimming
-        [InlineData( "-1'1\"", -1.08333333 )] // Without space
-        [InlineData( "-1′1″", -1.08333333 )] // Without space
-        [InlineData( "-1 ft 1 in", -1.08333333 )]
-        [InlineData( "-1ft 1in", -1.08333333 )]
+        [InlineData("-1'", 1)] // Feet only
+        [InlineData("-1′", 1)] // Feet only
```

Comment:
Wait, what? Shouldn't `expectedFeet` still be negative 1?

Reply:
Haha I re-copied-and-pasted. It's been a long day :) fixed